tls/s2n_connection.c

```diff
@@ -347,20 +345,25 @@ int s2n_connection_set_blinding(struct s2n_connection *conn, s2n_blinding blindi
     return 0;
 }
 
-int s2n_connection_get_delay(struct s2n_connection *conn)
+int64_t s2n_connection_get_delay(struct s2n_connection *conn)
 {
-    /* Delay between 1ms and 10 seconds in microseconds */
-    int min = 1000, max = 10 * 1000 * 1000;
+    /* Delay between 1ms and 10 seconds in nanoseconds */
+    int64_t min = 1000 * 1000, max = 10 * 1000 * 1000 * 1000;
```

Comment:
s2n_connection.c:351:55: error: integer overflow in expression [-Werror=overflow]

i'm guessing the ints are treated as that.  how about adding L after each constant integer?

also, consider making these static const